test/dotnet-new3.UnitTests/DotnetNewInstall.cs

```diff
@@ -157,6 +158,27 @@ public void CanInstallLocalNuGetPackage()
                 .And.HaveStdOutContaining("TestAssets.ConfigurationKitchenSink");
         }
 
+        [Fact]
+        public void CanPrintDebugOutputWhenInstalling()
+        {
+            new DotnetNewCommand(_log, "-i", "Microsoft.DotNet.Web.ProjectTemplates.5.0", "--quiet")
+                .WithCustomHive()
+                .WithWorkingDirectory(TestUtils.CreateTemporaryFolder())
+                .WithEnvironmentVariable("DOTNET_CLI_CONTEXT_VERBOSE", "true")
+                .Execute()
+                .Should()
+                .ExitWith(0)
+                .And
+                .NotHaveStdErr()
+                .And.NotHaveStdOutContaining("Determining projects to restore...")
+                .And.HaveStdOutContaining("The following template packages will be installed:")
+                .And.HaveStdOutMatching($"Success: Microsoft\\.DotNet\\.Web\\.ProjectTemplates\\.5\\.0::([\\d\\.a-z-])+ installed the following templates:")
+                .And.HaveStdOutContaining("web")
+                .And.HaveStdOutContaining("blazorwasm")
+                .And.HaveStdOutMatching("\\[\\d{4}\\-\\d{2}\\-\\d{2} \\d{2}:\\d{2}:\\d{2}\\.\\d{1,3}\\] " + Regex.Escape("[Debug] [Microsoft.TemplateEngine.Edge.Installers.NuGet.NuGetInstaller] => [Execute]: Microsoft.DotNet.Web.ProjectTemplates.5.0 is not a local NuGet package."))
```

Comment:
I would imagine that parameters overrides environment variable?

Reply:
not sure which parameters do you mean?

Reply:
I meant `--quiet` which is passed via CLI parameter...